README.md

```diff
@@ -68,6 +68,7 @@ Please make sure to include your changes in the Changelog below and consider cha
 
 ## Changelog
 - **UPCOMING**
+    -  Added Italian translations.Thanks to [@github4maninder](https://github.com/github4maninder)
```

Comment:
```suggestion
    -  Added Italian translations. Thanks to [@github4maninder](https://github.com/github4maninder)
```

Reply:
thank you for your suggestion. its fixed :)